### ISSUE TITLE
Spyder will not load due to utf8 encoding problem

### ISSUE THREAD

**Issue by PlokavianNerveGas** [spyder-ide/spyder#2605]
Hi,
I am unable to get spyder to load. I am running windows 7, 64 bit.

I have installed  Anaconda-2.3.0 64 bit and have attempted to run the bundled spyder.
I have also tried the latest separate spyder build and encoutered the exact same issue.

When running spyder via cmd I recieve the following error message:

``` python-traceback
Traceback (most recent call last):
  File "C:\Anaconda\scripts\spyder", line 2, in <module>
    from spyderlib import start_app
  File "C:\Anaconda\lib\site-packages\spyderlib\start_app.py", line 13, in <modu
le>
    from spyderlib.config import CONF
  File "C:\Anaconda\lib\site-packages\spyderlib\config.py", line 718, in <module
>
    subfolder=SUBFOLDER, backup=True, raw_mode=True)
  File "C:\Anaconda\lib\site-packages\spyderlib\userconfig.py", line 215, in __i
nit__
    self.load_from_ini()
  File "C:\Anaconda\lib\site-packages\spyderlib\userconfig.py", line 260, in loa
d_from_ini
    self.readfp(configfile)
  File "C:\Anaconda\lib\ConfigParser.py", line 324, in readfp
    self._read(fp, filename)
  File "C:\Anaconda\lib\ConfigParser.py", line 479, in _read
    line = fp.readline()
  File "C:\Anaconda\lib\codecs.py", line 678, in readline
    return self.reader.readline(size)
  File "C:\Anaconda\lib\codecs.py", line 533, in readline
    data = self.read(readsize, firstline=True)
  File "C:\Anaconda\lib\codecs.py", line 480, in read
    newchars, decodedbytes = self.decode(data, self.errors)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe8 in position 59: invalid
continuation byte
```

Is there some way to fix this?
Thanks

**Comment by ccordoba12** [comment]
The easiest solution is to change your user name so that it doesn't have non-ascii letters.

**Comment by gq9545** [comment]
I got a similar issue. I ran a file in Spyder but it encounters the _UnicodeDecodeError_. However, it works when I ran it in command line. Some details are as follows.   

I downloaded a Python [module](http://www.daimi.au.dk/~mailund/suffix_tree.html) that wraps a C implementation of a suffix tree. After built, it was ran in Spyder but I got the following error:

> runfile('F:/src/suffix_tree-2.1/build/lib.win32-2.7/suffix_tree.py',
> wdir='F:/src/suffix_tree-2.1/build/lib.win32-2.7')
> Traceback (most recent call last):
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\interactiveshell.py",
> line 3052, in run_code
>     self.showtraceback()
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\interactiveshell.py",
> line 1851, in showtraceback
>     value, tb, tb_offset=tb_offset)
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\ultratb.py",
> line 1240, in structured_traceback
>     self, etype, value, tb, tb_offset, number_of_lines_of_context)
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\ultratb.py",
> line 1157, in structured_traceback
>     self, etype, value, elist, tb_offset, number_of_lines_of_context
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\ultratb.py",
> line 511, in structured_traceback
>     lines = ''.join(self._format_exception_only(etype, value))
> 
>   File
> "G:\IDE\python\Anaconda\lib\site-packages\IPython\core\ultratb.py",
> line 623, in _format_exception_only
>     Colors.Normal, s))
> 
> UnicodeDecodeError: 'ascii' codec can't decode byte 0xb2 in position
> 20: ordinal not in range(128)

This error appears right after I import _suffix_tree.pyd, even before any other operations.

However, if I run the file in command line (python suffix_tree.py), it works successfully without any error.

PS: I use Windows and Spyder 2.3.5.2, and my user name are ASCII characters.

**Comment by rkresch** [comment]
Got the precisely same issue as PlokavianNerveGas, and my username is in ASCII. Appreciate any help.

**Comment by ccordoba12** [comment]
This should be fixed in Spyder 3.0.2.
